pkgs/development/tools/rust/cargo-ndk/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, glibc
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-ndk";
+  version = "2.12.6";
+
+  src = fetchFromGitHub {
+    owner = "bbqsrc";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-VGdFIMyZhb7SDWAXgs7kFlblYCO4rLf+3/N7Mashc4o=";
+  };
+
+  cargoHash = "sha256-pv6t9oKj5tdQjpvBgj/Y11uKJIaIWcaA9ib/Id/s+qs=";
+
+  buildInputs = lib.optionals stdenv.isLinux [ glibc ]
+    ++ lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks;[ CoreServices Security ]);
+  nativeBuildInputs = [ pkg-config ];
```

Comment:
Isn't glibc already part of stdenv? Also please order nativeBuildInputs before buildInputs for aesthetic reasons.

Reply:
I'm not sure if it is, but removing it doesn't break the build, so I don't think it's needed :)